pkg/es/config/config.go

```diff
@@ -301,15 +301,16 @@ func (c *Configuration) getConfigOptions(logger *zap.Logger) ([]elastic.ClientOp
 		}
 	} else {
 		httpTransport := &http.Transport{
-			Proxy: http.ProxyFromEnvironment,
+			Proxy:           http.ProxyFromEnvironment,
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: c.TLS.SkipHostVerify},
```

Comment:
when no TLS is involved at all, would non-nill `TLSClientConfig` cause issues?

Reply:
It should not godoc from `http.Transport.TLSClientConfig`

```
	// TLSClientConfig specifies the TLS configuration to use with
	// tls.Client.
	// If nil, the default configuration is used.
	// If non-nil, HTTP/2 support may not be enabled by default.
```

Reply:
this godoc doesn't seem to clarify what happens to tls / no-tls. Perhaps it's just driven off the http scheme